run-dljc.sh

```diff
@@ -100,179 +100,98 @@ fi
 
 ### Script
 
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
 # clone DLJC if it's not present
 if [ ! -d do-like-javac ]; then
     git clone https://github.com/kelloggm/do-like-javac
```

Comment:
Clone all scripts in siblings of SCRIPTDIR (that is, in SCRIPTDIR's parent).  Then, I don't think you need ORIGIN_PWD in this script.  That permits multiple executions of the script, in different directories, to use the same clone of scripts.